examples/cpp/example.cpp

```diff
@@ -369,40 +370,47 @@ int main(int argc, char **argv) {
     if (prefixLen > 0) { model.setPrefix(perfixSeq); }
 
     for (int i = 0; i < loop; ++i) {
+        secondIdCount = 0;
         model.config(/*maxLen*/ maxLen, /*numBeams*/ numBeams, /*numBeamHypsToKeep*/ 1, /*lenPenalty*/ 1.0,
                 /*doEarlyStopping*/ false, /*eosTokenId*/ -1, /*padTokenId*/ -1,
                 /*doSample*/ doSample, /*temperature*/ temperature,
                 /*topK*/ topK, /*topP*/ topP);
         model.input(input, batchSize);
 
         std::vector<int> firstIds;
-        std::vector<int> seconedIds;
+        std::vector<int> secondIds;
 
         if (!model.isDone()) {
             Timer t(isMaster, "[INFO] First token");
             firstIds = model.generate();
         }
 
+        Timer timerSecond;
         if (!model.isDone()) {
-            Timer t(isMaster, "[INFO] Second token");
-            seconedIds = model.generate();
+            secondIds = model.generate();
+            secondIdCount++;
         }
 
         if (isMaster && streamingOutput) {
             if (!firstIds.empty()) {
                 tokenizer->printResult(firstIds, batchSize, numBeams);
-                if (!seconedIds.empty()) { tokenizer->printResult(seconedIds, batchSize, numBeams); }
+                if (!secondIds.empty()) { tokenizer->printResult(secondIds, batchSize, numBeams); }
             }
         }
 
         while (!model.isDone()) {
             auto nextIds = model.generate();
+            secondIdCount++;
             if (isMaster && streamingOutput) { tokenizer->printResult(nextIds, batchSize, numBeams); }
         }
+        if (isMaster){
+            auto avgDuration = timerSecond.getTime() / float(secondIdCount);
```

Comment:
If output_len = 1, secondID Count will be 0.

Reply:
Good catch!
If the output token length is 1, then there will be no 'next token'!

Thanks